phone/src/os/apps/config/apps.tsx

```diff
@@ -127,7 +128,13 @@ export const APPS: IAppConfig[] = [
     backgroundColor: MATCH_APP_PRIMARY_COLOR,
     color: MATCH_APP_TEXT_COLOR,
     path: '/match',
-    Route: () => <AppRoute id="MATCH" path="/match" component={MatchApp} emitOnOpen={true} />,
+    Route: () => {
+      return (
+        <React.Suspense fallback={<LoadingSpinner />}>
+          <AppRoute id="MATCH" path="/match" component={MatchApp} emitOnOpen={true} />
+        </React.Suspense>
+      );
+    },
```

Comment:
Is this necessary here? Try and move the suspense farther down in the tree. This here would render a transparent background while suspending.

Reply:
Need `MatchApp` wrapped in a new component in that case. Can do.

Reply:
Or move the recoil value call in a component down the tree